README.md

```diff
@@ -15,4 +15,14 @@ npm install
 npm run [chrome, firefox]
 ```
 
+## Development (Chrome)
+
+``` bash
+# install development dependencies
+npm install 
+# compiles the Chrome extension to the `./test/chrome` directory
+npm run chrome:dev
```

Comment:
typo, dev:chrome

Reply:
Thanks for catching this.  I've fixed it and pushed it as another commit.  Thanks for the review and if you want I'll squash this commit if you want.